### DISCUSSION TITLE
Per App Volumes in Settings

### DISCUSSION THREAD

**Discussion by SpyderZT** [File-New-Project/EarTrumpet#1447] [Ideas]
Since there's obviously a list of per app volumes / devices in a config file somewhere once you customize them (As those settings carry over across app launches), it would be nice to have that list surfaced in the settings. Just a page listing out whatever apps have custom settings (With the ability to change the settings in said list being a nice to have as well).

If the app keeps track of Every app it sees, whether or not it has custom setting for them, That would be nice to see in the list as well. With perhaps the apps with custom settings at the top of the list (Though just being alphabetical order would be fine if that was a headache to set up).

**Comment by SpyderZT** [comment]
And just looking through other ideas, a checkbox for #1434 could be in that list as well, if the functionality to hide items exists.

**Comment by riverar** [comment]
Hey @SpyderZT, the OS handles remembering audio levels, not EarTrumpet. So there's nothing really for us to surface.

**Comment by SpyderZT** [comment]
Ahhhh... that's unfortunate. Is that list not something EarTrumpet can surface? I imagine this is Also a config file that can be seen, if not edited.
